auditbeat/docs/breaking.asciidoc

```diff
@@ -1,49 +1,4 @@
 [[auditbeat-breaking-changes]]
-== Breaking changes in 7.0
```

Comment:
These changes are already documented in libbeat docs, plus putting any new section here doesn't fit well with the 6.2 breaking changes hack in libbeat docs.